policy/files/table_test.go

```diff
@@ -25,45 +25,69 @@ import (
 func TestToTable(t *testing.T) {
 
 	tests := []struct {
-		name   string
-		input  string
-		output [][]string
+		name    string
+		input   string
+		output  [][]string
+		reverse string
 	}{
 		{
-			name:   "empty",
-			input:  "",
-			output: [][]string{},
+			name:    "empty",
+			input:   "",
+			output:  [][]string{},
+			reverse: "",
 		},
 		{
-			name:   "multiple empty rows",
-			input:  "\n     \n\n \n",
-			output: [][]string{},
+			name:    "multiple empty rows",
+			input:   "\n     \n\n \n",
+			output:  [][]string{},
+			reverse: "",
 		},
 		{
-			name:   "commented out row",
-			input:  "# this is a comment\n",
-			output: [][]string{},
+			name:    "commented out row",
+			input:   "# this is a comment\n",
+			output:  [][]string{},
+			reverse: "",
 		},
 		{
-			name:   "multiple rows with comment",
-			input:  "1 2 3\n 4 5#comment \n6 7 #comment\n 8",
-			output: [][]string{{"1", "2", "3"}, {"4", "5"}, {"6", "7"}, {"8"}},
+			name:    "field with spaces",
+			input:   "1 \"2 3\" 3\n",
+			output:  [][]string{{"1", "2 3", "3"}},
+			reverse: "1 '2 3' 3\n",
+		},
+		{
+			name:    "field with spaces",
+			input:   "1 'oidc:claims[\"https://example.com/my-custom-groups\"].contains(\"group with space and :\")' 3\n",
+			output:  [][]string{{"1", "oidc:claims[\"https://example.com/my-custom-groups\"].contains(\"group with space and :\")", "3"}},
+			reverse: "1 'oidc:claims[\"https://example.com/my-custom-groups\"].contains(\"group with space and :\")' 3\n",
+		},
+		{
+			name:    "multiple rows with comment",
+			input:   "1 2 3\n 4 5#comment \n6 7 #comment\n 8",
+			output:  [][]string{{"1", "2", "3"}, {"4", "5"}, {"6", "7"}, {"8"}},
+			reverse: "1 2 3\n4 5\n6 7\n8\n",
 		},
 		{
 			name: "realistic input",
 			input: `# Issuer Client-ID expiration-policy
-https://accounts.google.com 206584157355-7cbe4s640tvm7naoludob4ut1emii7sf.apps.googleusercontent.com 24h
-https://login.microsoftonline.com/9188040d-6c67-4c5b-b112-36a304b66dad/v2.0 096ce0a3-5e72-4da8-9c86-12924b294a01 24h`,
+		https://accounts.google.com 206584157355-7cbe4s640tvm7naoludob4ut1emii7sf.apps.googleusercontent.com 24h
```

Comment:
Because back tick is used, this will put a tab into the config file. I assume that tab is unintended?

Reply:
Fixex